pkg/config/mesh/mesh.go

```diff
@@ -91,7 +95,18 @@ func DefaultMeshConfig() meshconfig.MeshConfig {
 		EnableAutoMtls:                    &types.BoolValue{Value: false},
 		ThriftConfig:                      &meshconfig.MeshConfig_ThriftConfig{},
 		LocalityLbSetting:                 &v1alpha3.LocalityLoadBalancerSetting{},
+		ClusterLocalNamespaces:            append(make([]string, 0), defaultClusterLocalNamespaces...),
+	}
+}
+
+// IsClusterLocal indicates whether the given namespace is configured to be cluster-local.
+func IsClusterLocal(mesh *meshconfig.MeshConfig, namespace string) bool {
```

Comment:
How about moving this function  as a method of `model.Service`?
It is not clear to make it here and may cause misuse

Reply:
I added a utility method to Service that delegates to this.